trending_data/src/main/java/com/google/sps/servlets/Servlet.java

```diff
@@ -1 +1 @@
-public class Servlet {}
\ No newline at end of file
+public class Servlet {}
```

Comment:
Is there a reason for creating this file now?

Reply:
We created this file yesterday when we did the setup because we wanted to include the path to servlets in the directory structure